ocs_ci/deployment/aws.py

```diff
@@ -534,6 +534,13 @@ def create_rhel_instance(self):
                 InstanceId=inst_id,
             )
 
+    @retry(exceptions.CommandFailed, tries=10, delay=60, backoff=2)
```

Comment:
Do we really need to backoff=2? If I understand this parameter correctly it will double the delay each attempt. So if something will happen you will wait for a lot of time to fail.
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/retry.py#L16

I think that reasonable time to wait and retry will be about 5 max 10 mins WDYT?
so
```
@retry(exceptions.CommandFailed, tries=15, delay=30, backoff=1)
```
Sounds more reasonable to me

Reply:
Agree

Reply:
Done